codegen/src/main/java/software/amazon/awssdk/codegen/utils/PaginatorUtils.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2011-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *    http://aws.amazon.com/apache2.0
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.amazon.awssdk.codegen.utils;
+
+public class PaginatorUtils {
+
+    /**
+     * @param methodName Name of the method which returns only a single page response
+     * @return the name of the pagination enabled operation
+     *
+     */
+    public static String getSyncMethodName(String methodName) {
```

Comment:
Would this make more sense in NamingStrategy?

Reply:
DefaultNamingStrategy constructor is coupled to ServiceModel and CustomizationConfig. But the method doesn't require any of them. The classes where I use this method don't have enough information to create a NamingStrategy object.